hc-bar/js/bar.js

```diff
@@ -0,0 +1,338 @@
+// series is an array of objects
+//    name - eventType (Intercept, Strike, Other)
+//    data is an array of objects
+//      name - monthYear
+//      Y - # of event in m/y
+//      drilldown - monthYear-eventType
+
+// drilldown is an object
+//    allowPointDrilldown - false (drilldown shows events for all categories in that month/year)
+//    series is an array of objects
+//      name - eventType+Headline+Text (Strike, Intercept, Other)
+//      id - monthYear-eventType
+//      data is an array of arrays
+//        [day, # of events]
+
+var drilldownSeries = [];
+// Create empty array to push new event objects into
+var series = [];
+
+Highcharts.data({
+  // Load Data in from Google Sheets
+  googleSpreadsheetKey: "1AKM59m3iaOSSQgVsIoDcSWxjcpX781JIuBwB9YdDgFs",
+  googleSpreadsheetWorksheet: 1,
+  switchRowsAndColumns: true,
+  parsed: function parsed(columns) {
+    // set default values
+    var drilldown = "";
+    var dataObject = { other: {}, strike: {}, intercept: {} };
+    var dataArray = [];
+    var drilldownObject = {};
+    var event = "";
```

Comment:
I believe that this event object is never used in the code. Please remove it as its value is never read.